certified-connectors/Eagle/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2577 @@
+                  "description": "Is case-sensitive and must match the options for this Task Type",
+                  "title": "Phase"
+                },
+                "WORKCODE": {
+                  "type": "string",
+                  "description": "Must match EBMS work code exactly",
+                  "title": "Work code"
+                },
+                "MANAGER": {
+                  "type": "string",
+                  "description": "Manager",
+                  "title": "Manager",
+                  "x-ms-visibility": "advanced",
+                  "x-ms-dynamic-values": {
+                    "operationId": "GetTaskManager",
+                    "value-path": "MANAGER",
+                    "value-collection": "value",
+                    "value-title": "MANAGER",
+                    "parameters": {}
+                  }
+                },
+                "NOTE": {
+                  "type": "string",
+                  "description": "Note",
+                  "title": "Note"
+                },
+                "COMPLETED": {
+                  "type": "boolean",
+                  "enum": [
+                    true,
+                    false
+                  ],
+                  "description": "Yes or No",
+                  "title": "Completed"
+                },
+                "EXTERNALID": {
+                  "type": "string",
+                  "description": "Unique ID of the program integrated with EBMS",
+                  "title": "External ID",
+                  "x-ms-visibility": "advanced"
+                }
+              }
+            }
+          }
+        ],
+        "description": "This action will update an exising EBMS task (task module required)."
+      }
+    },
+    "/ARCUST": {
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Create a new customer",
+        "operationId": "CreateCustomer",
+        "x-ms-visibility": "important",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "TREE_ID": {
+                  "type": "string",
+                  "description": "Select the category where the customer is to be created",
+                  "title": "Customer Category",
+                  "x-ms-dynamic-values": {
+                    "operationId": "GetCustomerCategoryId",
+                    "value-path": "TREE_ID",
+                    "value-collection": "value",
+                    "value-title": "TREE_DESCR",
+                    "parameters": {}
+                  }
+                },
+                "ID": {
+                  "type": "string",
+                  "description": "Required unless the category is configured to created automatically. Must be unique.",
```

Comment:
@doniyorniazov please make sure that everything is grammatically correct and is free of typographical errors.